docs/contributing/features.md

```diff
@@ -0,0 +1,24 @@
+# New Features
+
+We review contributions of new features (e.g., new packages, new scripts, new flags) to the core stacks in an attempt to balance the value of the images to the Jupyter community with the cost of maintaining the images over time. Roughly speaking, we think about the following when considering additions:
+
+* *Usefulness to Jupyter users* - Is the feature generally applicable across domains? Does it work with Jupyter Notebook, JupyterLab, JupyterHub, etc.?
```

Comment:
I think this would be easier to read with **bold**:  (bold followed by unbolded colon)